OnLand/ControlLaptop/static/assets/js/FlightProfile.js

```diff
@@ -33,10 +74,25 @@ function setupConfigurationFormSubmit() {
       error: function handleErrorFormSubmission(data) {
         console.log("FAILED");
```

Comment:
More descriptive error

Reply:
If the error from the server is connection refused then we should relay that. Will be very important to know that we aren't connecting to the pod